selfdrive/car/subaru/values.py

```diff
@@ -7,7 +7,7 @@
 class CAR:
   ASCENT = "SUBARU ASCENT LIMITED 2019"
   IMPREZA = "SUBARU IMPREZA LIMITED 2019"
-  FORESTER = "SUBARU FORESTER 2019"
+  FORESTER = "SUBARU FORESTER 2019 - 2020"
```

Comment:
Don't change vehicle name

```suggestion
  FORESTER = "SUBARU FORESTER 2019"
```